### PR TITLE
First pass at cleaning out logstash web

### PR DESCRIPTION
**Pull request by untergeek**
All internal references, flags, kibana, vendor, everything must go!

If it referenced kibana or logstash-web, it is deprecated, and removed!

Operators are standing by...

### CODE REVIEW

File: docs/tutorials/10-minute-walkthrough/index.md [hunk 1]
```diff
@@ -107,18 +82,13 @@ Same as the previous step, but we'll output to ElasticSearch now.
 
 ### Run it
 
-    bin/logstash agent -f apache-elasticsearch.conf web
+    bin/logstash agent -f apache-elasticsearch.conf
```
Comment (suyograo) [inline docs/tutorials/10-minute-walkthrough/index.md]: we don't need agent anymore ..

Comment (untergeek) [inline docs/tutorials/10-minute-walkthrough/index.md]: Well, I _was_ only supposed to prune web and kibana, but I can remove agent, I suppose.

Also, isn't the 10-minute-walkthrough supposed to go the way of the dodo, too?  The non-asciidoc docs are all deprecated, aren't they?

Comment (suyograo) [inline docs/tutorials/10-minute-walkthrough/index.md]: yeah that caught my attention, so..

Lets keep this change only for ripping out web and open another PR for these things

**Comment by suyograo** [comment]
LGTM

**Comment by suyograo** [comment]
RIP `bin/logstash web`

**Pull request closed without merging**